src/repositories/car.js

```diff
@@ -0,0 +1,85 @@
+import models from '../models/';
+import { Op } from 'sequelize';
+
+export default {
+    getAllCars: () => {
+        return models.Car.findAll();
+    },
+    getCarsByStatusAndFuelLevel: (status, fuelLevel) => {
+        return models.Car.findAll({
+            where: {
+                status,
+                fuelLevel: {
+                    [Op.lt]: fuelLevel
+                }
+            }
+        })
+    },
+    getCarsByStatusAndNotAuthorized: (status) => {
+        return models.Car.findAll({
+            where: {
+                status,
+            },
+            attributes: ['vin', 'geoLatitude', 'geoLongitude'],
+            include: [
+                {
+                    model: models.Run,
+                    include: [
+                        {
+                            model: models.Driver,
+                            where: {
+                                creditCardId: null,
+                            },
+                            attributes: ['firstName', 'lastName', 'licenseNumber'],
+                        }
+                    ]
+                }
+            ],
+        });
+    },
+    getCarsByProducedDateAndMileage: (productionDate, mileage) => {
+        return models.Car.findAll({
+            where: {
+                productionDate: {
+                    [Op.lt]: productionDate,
+                },
+                mileage: {
+                    [Op.gt]: mileage,
+                }
+            }
+        });
+    },
+    getCarsByIdsAndStatus: (ids, status) => {
+        return models.Car.findAll({
+            where: {
+                id: {
+                    [Op.in]: ids,
+                },
+                status: {
+                    [Op.in]: status,
+                }
+            }
+        });
+    },
+
+    createCar: data => {
+        try {
+            return models.Car.create(data);
+        }
+        catch(err) {
+            throw err;
+        }
+    },
+    deleteCarBy: params => {
+        models.Car.destroy({
+            where: {
+                ...params
```

Comment:
First, good choise of truing to do it more flexible, but why only here? Second, does it work? I don't see any call proper call of this method